packages/xo-web/src/common/render-xo-item.js

```diff
@@ -167,6 +205,40 @@ export const SrResourceSetItem = [
 
 SrResourceSetItem.propTypes = XO_ITEM_PROP_TYPES
 
+@connectStore(() => {
+  // true to bypass view permissions
+  const getVdi = createGetObject()
+  const getSr = createGetObject((state, props) => {
+    const vdi = getVdi(state, props, true)
+    return vdi && vdi.$SR
+  })
+
+  return (state, props) => ({
+    sr: getSr(state, props),
```

Comment:
Aren't you missing a `true` here?

Reply:
No, in `getSr` I use `const vdi = getVdi(state, props, true)`